### DISCUSSION TITLE
org-cite-global-bibliography vs citar-bibliography

### DISCUSSION THREAD

**Discussion by maasdorp** [emacs-citar/citar#524] [Q&A] [answered]
Does one have to specify both  org-cite-global-bibliography and citar-bibliography?

If I only specify `(org-cite-global-bibliography '("~/bib/references.bib"))`, org-cite-insert returns a warning message about specifying citar-bibliography paths. When I specify both `(org-cite-global-bibliography '("~/bib/references.bib"))` and `(citar-bibliography '("/bib/references.bib"))` it works. Is this duplication necessary when I also specified  `(org-cite-insert-processor 'citar)`?

**Comment by bdarcus** [comment, marked as answer]
> Is this duplication necessary when I also specified `(org-cite-insert-processor 'citar)`?

It is now, but is not strictly necessary. 

I think at one point we had it so you didn't need to set `citar-bibliography` if you already set `org-cite-global-bibliography`.

WDYT @roshanshariff @localauthor?

**Reply by maasdorp** [reply]
Thanks, I see that the documentation under the org-cite heading also includes the citar-bibliography line. So all good! Thanks for your hard work.
